tsconfig.eslint.json

```
@@ -6,7 +6,7 @@
   },
   "extends": "./tsconfig.base.json",
   "include": [
-    "tools/**/*.ts",
+    "tools/**/*.{mts,ts}",
```

```suggestion
    "tools/**/*.ts",
    "tools/**/*.mts",
```

As far as I know, `include` can't handle brace expansions: https://github.com/microsoft/TypeScript/issues/47001

With `"tools/**/*.{mts,ts}"`:

```shell
$ yarn tsc -p tsconfig.eslint.json --showConfig | jq '.files'

[
  "./.eslintrc.js",
  "./jest.config.base.js",
  "./jest.config.js"
]
```

With `"tools/**/*.ts", "tools/**/*.mts"`:

```shell
$ yarn tsc -p tsconfig.eslint.json --showConfig | jq '.files'

[
  "./tools/dummypkg/index.d.ts",
  "./tools/release/release.mts",
  "./.eslintrc.js",
  "./jest.config.base.js",
  "./jest.config.js"
]
```

---

Good catch, thanks!